AnnoDesigner.Core/RecentFiles/RecentFilesSerializerInMemory.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.Core.RecentFiles
+{
+    public class RecentFilesSerializerInMemory : IRecentFilesSerializer
```

Comment:
A better name for this would be `RecentFilesMemorySerializer`. `Memory` is enough of an explanation, and `Serializer` is generally used as a suffix, rather than being used within the class name. (e.g `JsonSerializer`, `TextSerializer`).

Reply:
Therefore this is a draft 😀 
The name was just to reflect the interface, but I will adjust the names to your proposal.

Reply:
You can go with *InMemorySerializer if you want to. The main point of it is `Serializer` as the suffix.

Reply:
I also don't know if I can left off the `RcentFiles` because it is already in this namespace.
But maybe this could lead to confussion in classes that use this one.

So what is better?
1. `AnnoDesigner.Core.RecentFiles.RecentFilesInMemorySerializer`
2. `AnnoDesigner.Core.RecentFiles.InMemorySerializer`
3. something different

Reply:
I think you misunderstood - see the `*` at the start of my comment above? I meant you could go with `RecentFilesInMemorySerializer` instead of `RecentFilesMemorySerializer`, I was just too lazy to type it, so I wrote `*` instead.

Reply:
Nevermind, apparently I've misunderstood. Yeah, first one is better I think, or possibly `RecentFiles.Serializers`? Not too sure.